CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ option(BUILD_TESTING "Build tests" OFF)
 add_feature_info(BUILD_TESTING BUILD_TESTING "configure whether to build the test suite")
 include(CTest)
 
+# We're building SixtyFPS as a standalone package, not embedded into another project via FetchContent
+set(STANDALONE_BUILD ON)
```

Comment:
i guess it should be called SIXTYFPS_STANDALONE_BUILD

Reply:
It's just a variable, it won't end up in the cache (won't leak into user CMakeCache.txt). But I can rename it, sure.

Reply:
ah, good point, i thought it was used in api/sixtyfps-cpp/CMakeLists.txt by anyone using sixtyfps